Feature/code optimisation

ColonelMoutarde opened the pull request:
Juste de petites choses pour faire crier @sylvaner

Sylvaner reviewed (changes requested):
Mon IDE peut le faire automatiquement en 1 commit

Sylvaner committed:
PHPdoc and optims

PHPdoc, optims and return methodes

PHPdoc, optims and return methodes

PHPdoc

PHPdoc

PHPdoc and return values

Test if true is strict
